firmware/controllers/algo/engine.h

```diff
@@ -194,7 +199,9 @@ class EngineState {
 	float baroCorrection;
 
 	// speed density
-	float tChargeK;
+	float tCharge, tChargeK;
```

Comment:
Rate-of-change limiter is applied to degrees, so we store both Kelvin and degrees.